### ISSUE TITLE
[gatsby-remark-prismjs] inconsistent outputLines behavior

### ISSUE THREAD

**Issue by alexfornuto** [gatsbyjs/gatsby#19261]
## Description

I'm following the plugin [instructions](https://www.gatsbyjs.org/packages/gatsby-remark-prismjs/#shell-prompt) to designate output lines in a code block, but what's rendered doesn't match the lines I'm specifying.

### Steps to reproduce

Discovered using the [Pantheon Documentation](https://github.com/pantheon-systems/documentation) project, but  I've created a minified (as best I can) reproduction [here](https://github.com/alexfornuto/reproduce-outputlines-bug).

Define a code block as:

````
   ```bash{outputLines: 2-7,10}
   git clone git@github.com:pantheon-systems/documentation.git
   Cloning into 'documentation'...
   remote: Counting objects: 41601, done.
   remote: Compressing objects: 100% (137/137), done.
   remote: Total 41601 (delta 83), reused 0 (delta 0), pack-reused 41463
   Receiving objects: 100% (41601/41601), 112.21 MiB | 5.91 MiB/s, done.
   Resolving deltas: 100% (31995/31995), done.
   cd documentation/
   git checkout -b update-template
   Switched to a new branch 'update-template'
   ```
````

This matches the example provided on how to use this: ```bash{outputLines: 2-10,12} (from the README).

### Expected result

Prompts on all lines not specified (1, 8, and 9).

### Actual result

Rendered result from `gatsby develop`:
![image](https://user-images.githubusercontent.com/2349184/68156212-041c4800-ff11-11e9-8658-557a01ccaeee.png)

### Environment
From the minified environment:
```
  System:
    OS: Linux 5.3 Fedora 30 (Workstation Edition) 30 (Workstation Edition)
    CPU: (4) x64 Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
    Shell: 5.7.1 - /usr/bin/zsh
  Binaries:
    Node: 10.16.3 - /usr/bin/node
    npm: 6.9.0 - /usr/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 78.0.3904.70
    Firefox: 70.0
  npmPackages:
    gatsby: ^2.1.31 => 2.17.7 
    gatsby-image: ^2.0.33 => 2.2.30 
    gatsby-mdx: ^0.4.3 => 0.4.5 
    gatsby-plugin-manifest: ^2.0.24 => 2.2.26 
    gatsby-plugin-offline: ^2.0.25 => 2.2.10 
    gatsby-plugin-react-helmet: ^3.0.9 => 3.1.13 
    gatsby-plugin-sharp: ^2.0.28 => 2.2.36 
    gatsby-plugin-styled-components: ^3.0.7 => 3.1.11 
    gatsby-plugin-typescript: ^2.0.11 => 2.1.15 
    gatsby-remark-prismjs: ^3.2.5 => 3.3.20 
    gatsby-source-filesystem: ^2.0.24 => 2.1.35 
    gatsby-transformer-remark: ^2.3.2 => 2.6.32 
    gatsby-transformer-sharp: ^2.1.17 => 2.3.2 
  npmGlobalPackages:
    gatsby-cli: 2.8.5
```

**Comment by alexfornuto** [comment]
>You can also add the label "not stale" to keep this issue open!

No I can't. Only project members can add labels.

**Comment by alexfornuto** [comment]
Another example:

````
```bash{outputLines:2,8-33}
pwd
/Users/USERNAME/Sites/SITENAME
cd ..
pwd
/Users/USERNAME/Sites/
drush @pantheon.SITENAME.ENV status
 Drupal version : 7.26
 Site URI : ENV-SITENAME.pantheonsite.io
 Database driver : mysql
 Database hostname : 10.178.14.16
 Database username : pantheon
 Database name : pantheon
 Database : Connected
 Drupal bootstrap : Successful
 Drupal user : Anonymous
 Default theme : bartik
 Administration theme : seven
 PHP configuration : /srv/bindings/754cbef0a7b54a07ab07167ef8de7377/php5
                           3.ini
                           /srv/bindings/754cbef0a7b54a07ab07167ef8de7377/php5
                           3.ini
 Drush version : 5.10.1
 Drush configuration : /srv/bindings/754cbef0a7b54a07ab07167ef8de7377/drus
                           hrc.php
 Drupal root : /srv/bindings/754cbef0a7b54a07ab07167ef8de7377/code
 Site path : sites/default
 File directory path : sites/default/files
 Private file : sites/default/files/private
 directory path
 Temporary file : /srv/bindings/754cbef0a7b54a07ab07167ef8de7377/tmp
 directory path
```
````

Result:
![image](https://user-images.githubusercontent.com/2349184/69761444-04cd8600-112d-11ea-93ed-9e3d418f798e.png)


Note that the range excluded is not the same as the range defined, and the single line defined isn't excluded at all. This feature is not working at all as expected or documented.

**Comment by kissgyorgy** [comment]
I have the same issue

**Comment by alexfornuto** [comment]
> > You can also add the label "not stale" to keep this issue open!
> 
> No I can't. Only project members can add labels.

**Comment by alexfornuto** [comment]
Can someone with the rights to do so please add "not stale" to this issue?

**Comment by dan-mba** [comment]
Are you requiring prism-command-line.css in gatsby-browser.js?

Compare the HTML generated for your cases to the HTML in the prism documentation for 
[Command Line](https://prismjs.com/plugins/command-line/).

The command prompts are a collection of span tags with display set to block in a container floated to the left. Without the CSS it will not look right.

**Comment by alexfornuto** [comment]
> Are you requiring prism-command-line.css in gatsby-browser.js?

Yes, from `gatsby-browser.js`:

```
// Code block shell prompt
import "prismjs/plugins/command-line/prism-command-line.css"
```

The span tags are there and they have the CSS from prism and from my overriding CSS. I've tested using just the default CSS from the plugin, and the issue persists.

**Comment by dan-mba** [comment]
That output looks correct for `outputLines:2,8-33`

The output I think you want is `outputLines:2,5,7-33`

I built a codepen for your other case.
Does the output you are seeing look like the generated HTML(not the source HTML in the pen) 
[here](https://codepen.io/dan-mba/pen/povBPZB).

If not, can you paste in the full span with `class="command-line-prompt`.

**Comment by alexfornuto** [comment]
Thanks @dan-mba - while testing this I think I figured out the issue, to some extent. 

First, I was pulling the CSS to `gatsby-browser.js` from the plugin's default CSS file, then overriding it elsewhere. I'm now calling my overridden CSS directly, which I recreated because it was missing _something_ (probably [this](https://github.com/pantheon-systems/documentation/pull/5476/files#diff-2c79c851c9876585a48f0a06f4e661f7R35)) that was causing the issue.

@kissgyorgy if you're still having this issue, I suggest you reset your CSS to match what's in the default, then line-by-line recreate it to match your custom style.
